src/min.ts

```diff
@@ -0,0 +1,6 @@
+/* istanbul ignore next */
+import Component from '.';
+
+/* istanbul ignore next */
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+(window as any).BiotopeElement = Component;
```

Comment:
Discussion:
If this is approved, which name would be best? `Component`, `Element`, `BiotopeElement`? Something else?

Reply:
BioElement?

Reply:
Voting for `BiotopeElement` as it is the most explicit one